commons-library/pom.xml

```diff
@@ -28,10 +33,21 @@
             <version>1.18.2</version>
             <scope>provided</scope>
         </dependency>
+
         <dependency>
             <groupId>org.hibernate.javax.persistence</groupId>
             <artifactId>hibernate-jpa-2.1-api</artifactId>
             <version>1.0.2.Final</version>
```

Comment:
I recommend to move all versions of plugins and dependencies to properties section

Reply:
Thanks, those versions were changed to a properties as you suggested